var/spack/repos/builtin/packages/py-voxcell/package.py

```diff
@@ -25,3 +26,5 @@ class PyVoxcell(PythonPackage):
     depends_on('py-requests@2.18:', type='run')
     depends_on('py-scipy@0.13:', type='run')
     depends_on('py-six@1.0:', type='run')
+
+    depends_on('libsonata+python@develop', type='run')
```

Comment:
to be replaced with `py-libsonata` once / if #350 is merged.

Reply:
@arsenius7 : Currently `libsonata@develop` is not building (afaik).

Reply:
> to be replaced with `py-libsonata` once / if #350 is merged.

At #350 we reverted to `libsonata+python` for the moment.